index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="amen">
+    <header class="navItems">
+     
+        
+        <div class="images">
+         <div class="space">
+            <div class="image01"><img src="assets/Shape.png" alt="#" class="shape00"></div>
+            <div class="image02"><img src="assets/Shape_1.png" alt="#"  class="shape01"></div>
+            </div>
+            <div class="menuitem">
+             <div class="boxic"><img src="assets/ic_menu.svg"  alt="#" class="ic_menu"></div>  
+               </div>
+
+            </div>
+    </header>
+
+  <section class="container">
+       
+    <div class=" box name"><h3>Samuel Ackon</h3></div>
+    <div class=" box intro"><h4>Hello! I am a software developer! 
```

Comment:
- [ ] [REQUIRED] As requested by the previous reviewer,  please change the tag of this paragraph to a p tag instead of a div tag because p tags enclose paragraphs.